egs/librispeech/ASR/transducer/decode.py

```diff
@@ -401,7 +398,6 @@ def main():
 
     # <blk> and <sos/eos> are defined in local/train_bpe_model.py
     params.blank_id = sp.piece_to_id("<blk>")
```

Comment:
So is the idea that we use blank id for sos and eos as well, in the decoder?
BTW, I think the comment a line above may be wrong.

Reply:
> So is the idea that we use blank id for sos and eos as well, in the decoder?

I think there is only one extra blank symbol in transducer. There are no SOS and EOS.

---

> BTW, I think the comment a line above may be wrong.

Yes, you're right. I am fixing it.